Add Current and next element

Brookke opened the pull request.

Commit:
Add current and next element

Brookke commented:
Would someone be able to review this please.

LordAro commented on views/elements/current_and_next.tmpl:
```diff
@@ -0,0 +1,20 @@
+{{define "current_and_next"}}
```
What's your reasoning for making this a separate file? It leaves index.tmpl rather small

Brookke replied:
It means we can use it on other pages, and makes modifications simpler in the future.

LordAro commented on views/elements/current_and_next.tmpl:
```diff
@@ -0,0 +1,20 @@
+{{define "current_and_next"}}
+
+	{{if .Current}}
```
Lots of whitespace here, which is not being removed ( https://golang.org/doc/go1.6#template ) perhaps some can be trimmed? or stripped accordingly?

ChrisTheBaron replied:
My bad, yes all that whitespace can be removed. I didn't know about the trim functionality :stuck_out_tongue:

LordAro commented on views/elements/current_and_next.tmpl:
```diff
@@ -0,0 +1,20 @@
+{{define "current_and_next"}}
+
+	{{if .Current}}
```
Also, tabs for indenting templates? Not sure about that

Brookke replied:
I was just following the rest of the templates with this or is it specific to elements?

ChrisTheBaron replied:
@LordAro what would be wrong with using tabs here?

Brookke replied:
So I can't find it in the docs, but from some of the example files I've seen, it seems two spaces for the .tmpl files might be best and tabs for the .go files

LordAro commented on views/elements/current_and_next.tmpl:
```diff
@@ -0,0 +1,20 @@
+	{{.StartTime.Format "15:04"}} - {{.EndTime.Format "15:04"}}
+</p>
+
+{{end}}
```
no trailing newline :(

ChrisTheBaron replied:
Again, my bad. I think I wrote most of this code originally.

LordAro commented on views/index.tmpl:
```diff
@@ -1,30 +1,11 @@
-{{end}}
-{{end}}
+	{{end}}
+{{end}}
```
no trailing newline :(

Brookke committed:
Add trailing new line

Brookke committed:
Removed some whitespace

Can we merge up until now, then go through in a separate PR and tidy up
all the template files?
